src/main/java/uk/gov/justice/digital/oasys/jpa/repository/SimpleAssessmentRepository.java

```diff
@@ -30,7 +30,7 @@ public SimpleAssessmentRepository(EntityManager entityManager) {
     public Optional<Assessment> getLatestAssessment(Long offenderId, String filterGroupStatus, String filterAssessmentType, Boolean filterVoided, String filterAssessmentStatus) {
         var query = getAssessmentsQueryForOffender(offenderId);
         filterQuery(query, filterGroupStatus, filterAssessmentType, filterVoided, filterAssessmentStatus);
-        var assessment = query.orderBy(QAssessment.assessment.dateCompleted.desc()).fetchFirst();
+        var assessment = query.orderBy(QAssessment.assessment.createDate.desc()).fetchFirst();
```

Comment:
I'm aware of a discussion around this, don't different clients want different things?